traffic_portal/app/src/common/modules/form/user/FormUserController.js

```diff
@@ -50,7 +50,7 @@ var FormUserController = function(user, $scope, $location, formUtils, stringUtil
     $scope.labelize = stringUtils.labelize;
 
     $scope.viewDeliveryServices = function() {
-        $location.path($location.path() + '/delivery-services');
+        $location.path('/tenants/' + userModel.user.tenantId + '/delivery-services');
```

Comment:
So what this does is it directs you to the logged in user's tenant delivery services page every time which is not what you want. You want this to direct you to the user that was clicked's delivery services page. 

For example, if i'm on the this page: https://tp.domain.com/#!/users/244, then clicking "view delivery services" should direct me to `$location.path('/tenants/' + user[244].tenantId + '/delivery-services');`

Reply:
actually, i think you an just get rid of the `userModel` part.....or you can pass user into the viewDeliveryServices function if you want to be more explicit.